app/src/main/java/com/kks/nimblesurveyjetpackcompose/util/ReleaseTree.kt

```diff
@@ -0,0 +1,13 @@
+package com.kks.nimblesurveyjetpackcompose.util
+
+import android.util.Log
+import org.jetbrains.annotations.NotNull
+import timber.log.Timber
+
+class ReleaseTree : @NotNull Timber.Tree() {
+    override fun log(priority: Int, tag: String?, message: String, t: Throwable?) {
+        if (priority == Log.ERROR || priority == Log.WARN) {
+            // SEND ERROR REPORTS TO YOUR Crashlytics.
```

Comment:
If we're gonna implement crashlytics, we should add a `TODO` here. Otherwise we probably dont need this class 😉